server/api/reviews.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const router = express.Router();
+const Reviews = require("../database/models/Review");
+const Bots = require("../database/models/Bot");
+const Review = require("../database/models/Review");
+
+// Post user review -- requires Oauth to actually function --
+router.post("/reviews/:id", async (req, res) => {
+
+    const { id } = req.params;
+    const { userID, review } = req.body;
+
+    // Check if the bot exists
+    const foundBot = await Bots.findOne({ id });
+    if (!foundBot) return res.status(404).json({ message: "That bot doesn't exist in the database.", error: "Not Found."});
+
+    // Check if the user already reviewed this bot
+    const userReviewd = await Reviews.findOne({ botId: id, userID });
```

Comment:
It's called userId in the database so i am gonna update it because it is userId now not userID

Reply:
Yeah, I thought I had changed all of them, but I guess I didn't, thats fine

Reply:
```js
else return res.status(200).json({ message: "Successfully deleted the review from the database." });
``` 
Are u actually deleting the thing?